core/controlplane/config/templates/cloud-config-controller

```diff
@@ -2711,14 +2743,31 @@ write_files:
     content: {{.AssetsConfig.EtcdTrustedCA}}
 {{ end }}
 
+  - path: /etc/kubernetes/proxy-kubeconfig.yaml
```

Comment:
doesn't it go by default to localhost:8080 by default?

Reply:
ignore me, it is a daemonset now, and you need `--kubeconfig` file on every pod even on controllers